chords/op02n12a.chords.tsv

```diff
@@ -1,2 +1,139 @@
+1	1	1	1	1.5	1/4	1/4	Chord	3/4	1	1	3/8	1/4	3/2	0									
+1	1	5/2	5/2	0.5	5/8	5/8	Chord	3/4	1	1	1/8	1/8	1	1									
+1	1	0	0	1.0	0	0	Chord	3/4	3	1	1/4	1/4	1	2									
+1	1	1	1	2.0	1/4	1/4	Chord	3/4	3	1	1/2	1/2	1	3									
+2	2	3	3	1.0	0	0	Chord	3/4	1	1	1/4	1/4	1	4									
+2	2	4	4	1.0	1/4	1/4	Chord	3/4	1	1	1/4	1/4	1	5									
+2	2	5	5	1.0	1/2	1/2	Chord	3/4	1	1	1/4	1/4	1	6									
+2	2	3	3	0.0	0	0	FiguredBass	3/4	3	1	0									3/4	6		
+2	2	3	3	3.0	0	0	Chord	3/4	3	1	3/4	1/2	3/2	7									
+3	3	6	6	1.0	0	0	Chord	3/4	1	1	1/4	1/4	1	8									
+3	3	7	7	2.0	1/4	1/4	Chord	3/4	1	1	1/2	1/2	1	9									
+3	3	6	6	1.0	0	0	Chord	3/4	3	1	1/4	1/4	1	10									
+3	3	7	7	0.0	1/4	1/4	FiguredBass	3/4	3	1	0									3/8	6		
+3	3	7	7	1.5	1/4	1/4	Chord	3/4	3	1	3/8	1/4	3/2	11									
+3	3	17/2	17/2	0.5	5/8	5/8	Chord	3/4	3	1	1/8	1/8	1	12									
+4	4	9	9	1.0	0	0	Chord	3/4	1	1	1/4	1/4	1	13									
+4	4	10	10	1.5	1/4	1/4	Chord	3/4	1	1	3/8	1/4	3/2	14									
+4	4	23/2	23/2	0.25	5/8	5/8	Chord	3/4	1	1	1/16	1/16	1	15									
+4	4	47/4	47/4	0.25	11/16	11/16	Chord	3/4	1	1	1/16	1/16	1	16									
+4	4	9	9	0.0	0	0	FiguredBass	3/4	3	1	0									1/4	4		
+4	4	9	9	1.0	0	0	Chord	3/4	3	1	1/4	1/4	1	17									
+4	4	10	10	0.0	1/4	1/4	FiguredBass	3/4	3	1	0									1/4	#		
+4	4	10	10	1.0	1/4	1/4	Chord	3/4	3	1	1/4	1/4	1	18									
+4	4	11	11	1.0	1/2	1/2	Chord	3/4	3	1	1/4	1/4	1	19									
+5	5	12	12	3.0	0	0	Chord	3/4	1	1	3/4	1/2	3/2	20									
+5	5	13	13	1.5	1/4	1/4	Chord	3/4	2	1	3/8	1/4	3/2	21									
+5	5	29/2	29/2	0.5	5/8	5/8	Chord	3/4	2	1	1/8	1/8	1	22									
+5	5	12	12	1.0	0	0	Chord	3/4	3	1	1/4	1/4	1	23									
+5	5	13	13	2.0	1/4	1/4	Chord	3/4	3	1	1/2	1/2	1	24									
+6	6	16	16	1.5	1/4	1/4	Chord	3/4	1	1	3/8	1/4	3/2	25									
+6	6	35/2	35/2	0.5	5/8	5/8	Chord	3/4	1	1	1/8	1/8	1	26									
+6	6	15	15	1.0	0	0	Chord	3/4	2	1	1/4	1/4	1	27									
+6	6	16	16	1.0	1/4	1/4	Chord	3/4	2	1	1/4	1/4	1	28									
+6	6	17	17	1.0	1/2	1/2	Chord	3/4	2	1	1/4	1/4	1	29									
+6	6	15	15	0.0	0	0	FiguredBass	3/4	3	1	0									3/4	6		
+6	6	15	15	3.0	0	0	Chord	3/4	3	1	3/4	1/2	3/2	30									
+7	7	18	18	1.0	0	0	Chord	3/4	1	1	1/4	1/4	1	31									
+7	7	19	19	1.0	1/4	1/4	Chord	3/4	1	1	1/4	1/4	1	32									
+7	7	20	20	1.0	1/2	1/2	Chord	3/4	1	1	1/4	1/4	1	33									
+7	7	18	18	1.0	0	0	Chord	3/4	2	1	1/4	1/4	1	34									
+7	7	19	19	2.0	1/4	1/4	Chord	3/4	2	1	1/2	1/2	1	35									
+7	7	18	18	0.0	0	0	FiguredBass	3/4	3	1	0									1/2	6		
+7	7	18	18	2.0	0	0	Chord	3/4	3	1	1/2	1/2	1	36									
+7	7	20	20	0.0	1/2	1/2	FiguredBass	3/4	3	1	0									1/4	6		
+7	7	20	20	1.0	1/2	1/2	Chord	3/4	3	1	1/4	1/4	1	37									
+8	8	21	21	0.5	0	0	Chord	3/4	1	1	1/8	1/8	1	38									0
+8	8	43/2	43/2	0.5	1/8	1/8	Chord	3/4	1	1	1/8	1/8	1	39									
+8	8	22	22	1.5	1/4	1/4	Chord	3/4	1	1	3/8	1/4	3/2	40									
+8	8	47/2	47/2	0.5	5/8	5/8	Chord	3/4	1	1	1/8	1/8	1	41									
+8	8	21	21	1.0	0	0	Chord	3/4	2	1	1/4	1/4	1	42									
+8	8	22	22	1.5	1/4	1/4	Chord	3/4	2	1	3/8	1/4	3/2	43	ornamentPrecompMordentUpperPrefix								
```

Comment:
@johentsch After checking this now I see this. Adding ornaments adds it into the chords.tsv ....should we refrain from adding ornaments?